lib/el3_runtime/aarch32/cpu_data.S

```diff
@@ -34,7 +34,13 @@ endfunc _cpu_data
  * -----------------------------------------------------------------
  */
 func _cpu_data_by_index
+#if CRASH_REPORTING
+	mov	r1, #(CACHE_WRITEBACK_GRANULE * 2)
```

Comment:
The only concern is if the size of cpu_data_t changes, the corresponding will not reflect here. How about defining a new CPU_DATA_SIZE macro in the header which also allows to get rid of conditional compilation here?